### DISCUSSION TITLE
Saving/Loading UVs via Streaming USD

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#1707] [Q&A] [answered]
I am using Maya 2022.1 - Vanilla, with the built in 0.10.0 build of mayaUsd.

I am having an issue seeing UVs in Maya when streaming in a USD asset via:
`Create->Universal Scene Description(USD)->Stage From File`

**To Reproduce:**

1. I am creating a standard Maya Polygon Cube, which automatically includes a standard unwrapping on the box shown in the UV Editor.
![image](https://user-images.githubusercontent.com/69012021/133688595-81bae8a1-8c23-42ef-9571-caf5ab518aa2.png)
2. Use the File->Export Selection menu with the standard options to export the cube somewhere. 
![image](https://user-images.githubusercontent.com/69012021/133688412-66d77247-5357-426a-a915-4386ab8f837a.png)
3. Use the menu item: `Create->Universal Scene Description(USD)->Stage From File` to load the cube as a USD asset.
4. When selecting the root node of the USD asset, as well as the mesh which should contain the UVs, no UVs are shown in the UV Editor window.
![image](https://user-images.githubusercontent.com/69012021/133688681-71c8abb3-cb29-462f-9d65-b66756f45a18.png)

Do UVs only get loaded in when using the `File->Import` menu item to load? This seems to be the case for me in my testing.
I know this method translates the USD data back into native Maya data, and I would like to avoid this if possible.

**Just to confirm:**
This does appear to behave the same on the latest 0.12.0 release as well. 

Thanks for the help.

**Comment by santosd** [comment, marked as answer]
Hello @brycegbrazen,

Thank you for posting on our discussions and for asking this question. The reason you are not seeing the UV's from a USD prim is because at this moment there is no support for Maya to see that information so there is no workflow for editing and seeing UV's. This is something that is being looked into for the future.

I hope that information is helpful in answering your question.

Cheers,
David Santos

**Reply by brycegbrazen** [reply]
@santosd Thank you so much for the response! I appreciate it! Is there a list somewhere that outlines the USD features that are currently supported by Maya vs the features that are in progress or are in the backlog?
